ElvargServer/src/main/java/com/elvarg/game/entity/impl/player/persistence/dynamodb/DynamoDBPersistence.java

```diff
@@ -1,24 +1,28 @@
-package com.elvarg.game.entity.impl.player;
+package com.elvarg.game.entity.impl.player.persistence.dynamodb;
 
+import com.elvarg.game.entity.impl.player.Player;
+import com.elvarg.game.entity.impl.player.persistence.PersistenceMethod;
+import com.elvarg.game.entity.impl.player.persistence.PlayerSave;
 import com.elvarg.game.entity.impl.playerbot.PlayerBot;
+import software.amazon.awssdk.enhanced.dynamodb.DynamoDbEnhancedClient;
 import software.amazon.awssdk.enhanced.dynamodb.DynamoDbTable;
 import software.amazon.awssdk.enhanced.dynamodb.Key;
 import software.amazon.awssdk.enhanced.dynamodb.TableSchema;
 import software.amazon.awssdk.regions.Region;
 import software.amazon.awssdk.services.dynamodb.DynamoDbClient;
-import software.amazon.awssdk.enhanced.dynamodb.DynamoDbEnhancedClient;
 
 import java.time.Instant;
 
-public class PlayerSaveDb {
+public class DynamoDBPersistence extends PersistenceMethod {
 
     private static DynamoDbClient dynamoDbClient = DynamoDbClient.builder().region(Region.EU_WEST_1).build();
     private static DynamoDbEnhancedClient enhancedClient = DynamoDbEnhancedClient.builder().dynamoDbClient(dynamoDbClient).build();
     private static String playerTableName = System.getenv("PLAYER_TABLE_NAME");
 
     private static final TableSchema<PlayerSaveRecord> PLAYER_SAVE_TABLE_SCHEMA = TableSchema.fromClass(PlayerSaveRecord.class);
 
-    public static void save(Player player) {
+    @Override
+    public void save(Player player) {
         if (playerTableName == null || player instanceof PlayerBot) {
             return;
         }
```

Comment:
I would remove the check for if table is passed now
```suggestion
        if (player instanceof PlayerBot) {
            return;
        }
```